client/main.jsx

```diff
@@ -218,6 +219,210 @@ const ReactContainer = createContainer(() => {
+  },
+  updated() {
+    logTime();
+  },
+};
+
+const VueRecursive = {
+  name: 'recursive',
+  functional: true,
+  props: ['depth'],
+  render(h, { props }) {
+    let content;
+    if(props.depth) {
+      content = [
+        h(VueRecursive, {
+          props: {
+            depth: props.depth - 1,
+          },
+          key: 1,
+        }),
+        h(VueRecursive, {
+          props: {
+            depth: props.depth - 1,
+          },
+          key: 2,
+        }),
+        h(VueRecursive, {
+          props: {
+            depth: props.depth - 1,
+          },
+          key: 3,
+        }),
+        h(VueRecursive, {
+          props: {
+            depth: props.depth - 1,
+          },
+          key: 4,
+        }),
+        h(VueRecursive, {
+          props: {
+            depth: props.depth - 1,
+          },
+          key: 5,
+        }),
+      ];
+    } else {
+      content = h('button', {
+        on: {
+          click() {
+            console.log('Clicked!');
+          },
+        },
+      }, 'Bottom!');
+    }
+    return h('ul', [
+      h('li', [
+        h('p', `Depth ${props.depth}`),
+        content,
+      ])
+    ])
+  },
+};
+
+const VueRecursiveBase = {
+  name: 'recursive-base',
+  render(h) {
+    return h(VueRecursive, {
+      props: {
+        depth: 6,
+      },
+    });
+  },
+  mounted() {
+    logTime();
+  },
+  updated() {
+    logTime();
+  },
+};
+
+
+
+let vue;
+
+Template.vueContent.onRendered(function() {
+  vue = new Vue({
+    name: 'vue-base',
+    el: '#vue',
+    render(h) {
+      if(this.type === 'other') {
+        return h(VueOther);
+      } else if(this.type === 'recursive') {
+        return h(VueRecursiveBase);
+      } else if(this.type === 'table') {
+        return h(VueTable, {
+          props: {
+            content: this.content,
+          },
+        });
+      } else {
+        return h('p', 'Wrong type');
+      }
+    },
+    meteor: {
+      type() {
+        const selector = contentSelector.get();
+        if(selector === 'othervue') {
+          return 'other';
+        } else if(selector === 'recursivevue') {
+          return 'recursive';
+        } else {
+          return 'table';
+        }
+      },
+      content() {
+        const selector = contentSelector.get();
+        let collection;
+        if (selector === 'table1vue') {
+          collection = collection1;
+        } else if (selector === 'table2vue') {
+          collection = collection2;
+        } else if (selector === 'table3vue') {
+          collection = collection3;
+        } else {
+          return null;
+        }
+        // Optimization of the Meteor data with Object.freeze
+        // because we know it's the source of truth
+        // and we don't need vue to setup reactivity on every
+        // nested object, Tracker will take care of the reactivity
+        // This increase speed ~2 times
+        return Object.freeze(collection.find({}, {sort: {order: 1}}).fetch());
```

Comment:
You mean Tracker integration.

But this could be detected automatically. Just add a weak dependency on Tracker, and if `Tracker.isActive` then you disable Vue's reactivity.

Can you reference that discussion?